cmd/account/federation-policy/federation-policy.go

```diff
@@ -110,6 +110,8 @@ func newCreate() *cobra.Command {
 	// TODO: short flags
 	cmd.Flags().Var(&createJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
+	// TODO: complex arg: policy
```

Comment:
The TODO's seem to pertain to the body field itself. Any chance we can omit them?